packages/core/admin/admin/src/pages/InternalErrorPage.tsx

```diff
@@ -26,7 +27,7 @@ export const InternalErrorPage = () => {
       <ContentLayout>
         <EmptyStateLayout
           action={
-            <LinkButton variant="secondary" endIcon={<ArrowRight />} to="/">
+            <LinkButton variant="secondary" endIcon={<ArrowRight />} href="/">
```

Comment:
How do we handle typescript not knowing the `to` porp?

Reply:
ts-expect-error directive. It's everywhere at the moment. We have some ideas how to solve this though... in the future...

Reply:
![Screenshot 2024-04-08 at 14 07 05](https://github.com/strapi/strapi/assets/16720378/3af3edc7-320e-4f80-8186-5477bd0809b6)
Doing that the style automatically changes to this

Reply:
Same, I don't really understand the point of converting them to something and put the ts-expect-error having this solution that works. Is it conceptually so different? What am I missing here?

Reply:
Taken from [the docs](https://reactrouter.com/en/main/components/link)

> A <Link> is an element that lets the user navigate to another page by clicking or tapping on it. In react-router-dom, a <Link> renders an accessible <a> element with a real href that points to the resource it's linking to. This means that things like right-clicking a <Link> work as you'd expect. You can use <Link reloadDocument> to skip client side routing and let the browser handle the transition normally (as if it were an <a href>).

so in short, from my understanding if you don't have it, it'll do a full "browser" navigation instead of a client navigation the former being similar to a reload of the page? But i've not tested it lol